### ISSUE TITLE
Changing Node transformation breaks Node::getRotation(float&, float&, float&, ERotationConvention&) call

### ISSUE THREAD

**Issue by ParadoxDaKo** [GENIVI/ramses#50]
**Used Ramses version:** 27.0.5
**Replication Steps:**
1. Create a Node.
2. Move this Node somewhere else via ```setTranslation()```.
3. Get rotation data from this Node via ```getRotation(float& x, float& y, float& z, ERotationConvention& rotationConvention)```.
4. This ```getRotation()``` call will not return statusOK and all the values of all four output variables will not be changed, thus hindering further work with this Node.

Strangely, using the ```getRotation(float&, float&, float&)``` variant returns valid rotation data.

_Edit:_ This bug only seems to occur on Windows.

Here is a GTest that replicates this problem:

```cpp
TEST(ramses, node_TransformationDoesNotInvalidateRotationData) {
	ramses::RamsesFramework ramsesFramework;
	ramses::RamsesClient& client = *ramsesFramework.createClient("example client");
	ramses::Scene* scene = client.createScene(ramses::sceneId_t(123u), ramses::SceneConfig(), "example scene");
	ramses::Node* node = scene->createNode("desc");

	{
		float x, y, z;
		ramses::ERotationConvention convention;
		// getRotation() with convention should properly initialize variables before moving Node
		auto rotationStatus = node->getRotation(x, y, z, convention);

		ASSERT_EQ(rotationStatus, ramses::StatusOK);
		ASSERT_EQ(x, 0);
		ASSERT_EQ(y, 0);
		ASSERT_EQ(z, 0);
		ASSERT_EQ(convention, ramses::ERotationConvention::XYZ);
	}
	
	node->setTranslation(2, 2, 2);

	{
		float x, y, z;
		ramses::ERotationConvention convention;
		// getRotation() with convention should properly initialize variables after moving Node
		auto rotationStatus = node->getRotation(x, y, z, convention);

		ASSERT_EQ(rotationStatus, ramses::StatusOK);
		ASSERT_EQ(x, 0);
		ASSERT_EQ(y, 0);
		ASSERT_EQ(z, 0);
		ASSERT_EQ(convention, ramses::ERotationConvention::XYZ);
	}
}

**Comment by ParadoxDaKo** [comment]
Addendum: This bug only seems to occur on Windows.

**Comment by tohammer** [comment]
Thanks for the report! The issue is understood and we are working on a fix. Current plan is to ship it with next release

**Comment by tohammer** [comment]
That issue should be fixed with latest 27.0.101 release.
